### ISSUE TITLE
Nicovideo geoblocking question

### ISSUE THREAD

**Issue by trejkaz** [rg3/youtube-dl#16400]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`)
- Use the *Preview* tab to see what your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2018.05.01*. If it's not, read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2018.05.01**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
- [x] Checked that provided video/audio/playlist URLs (if any) are alive and playable in a browser

### What is the purpose of your *issue*?
- [ ] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [x] Question
- [ ] Other

---

### The following sections concretize particular purposed issues, you can erase any section (the contents between triple ---) not applicable to your *issue*

---

This URL: http://www.nicovideo.jp/watch/1503475085

Gives the usual error for Japan-only videos even in a browser, which is why this isn't a bug report. When using youtube-dl, I get:

    ERROR: niconico reports error: domestic_video

If I provide credentials, I get this instead:

    ERROR: Unable to download webpage: HTTP Error 403: Forbidden (caused by HTTPError()); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

I don't think this inconsistency is necessarily an issue, but my actual problem is that no matter what I have tried to get around this blocking, it doesn't seem to work. So I'm wondering if anyone knows exactly what kind of geoblocking nico is using.

Tried so far:

* Connecting through web proxy running in Linode Tokyo 2
* Connecting through a Cisco-IPSEC VPN running in Linode Tokyo 2
* Running youtube-dl itself from the shell of a node running in Linode Tokyo 2
* Editing my user account info to say I'm from Japan

Surely the issue isn't that they have realised people use VPNs and blocked all networks which people use for hosting servers as well? That would ruin things for everyone.

**Comment by hakanai** [comment]
I will take it that the answer is nobody knows.

**Comment by dirkf** [comment]
No, the answer is that the ancient question is about an ancient version. If you were to determine that the question is still relevant for the latest nightly version (#30839), someone might investigate. I have no idea whether Nicovideo extraction, which was fixed since this post, still works.

>Surely the issue isn't that they have realised people use VPNs and blocked all networks which people use for hosting servers as well? 

Sadly, this is even more likely to be the case now.
